src/traces/sunburst/plot.js

```diff
@@ -248,26 +253,29 @@ function plotOne(gd, cd, element, transitionOpts) {
             s.attr('data-notex', 1);
         });
 
+        var font = Lib.extendFlat({}, helpers.determineTextFont(trace, pt, fullLayout.font), {});
```

Comment:
(duplicate comment about strange `Lib.extendFlat` call)

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/4420/commits/a27dc29ca87303fd53bf994bfe1203b3c5e96887.